### ISSUE TITLE
Arches Resource Editor: Editor does not list tree nodes in same order as model

### ISSUE THREAD

**Issue by dwuthrich** [archesproject/arches#3670]
Arches new graph editor allows designers to order branches, but the new resource editor does not render the tree in the same order.  This means that the designers decision about how to order branches is not shown in the editor.

**Comment by dwuthrich** [comment]
@robgaston Right, it makes sense that the editor currently uses the sort order in the card manager.  Given our new approach to building graphs, maybe we should unify the ordering in the graph manager and card manager.

**Comment by robgaston** [comment]
@apeters @dwuthrich Is this ticket still on hold?  We've not yet implemented sortability in the new card manager, and I was going to add that in, but it seems to me that if we are going to unify on a single sorting scheme then we should take that into account when implementing sorting in the card manager. (we don't currently have a ticket for supporting sorting in the card manager)

thoughts?

**Comment by apeters** [comment]
I think the issue is that we didn't budget for unifying sorting across the graph tree and card tree.

**Comment by robgaston** [comment]
@apeters I understand that, I'm just suggesting that there is outstanding work around sorting in the card manager tree, which is currently not implemented.  we need to implement sorting in the card manager tree to complete work there.

my question is this: if we are going to need to work on sorting in the card manager, should we just take this ticket on instead?

**Comment by dwuthrich** [comment]
@robgaston @apeters v4.3 should support ordering of nodes within a card in the Card Manager, and order of node groups in the Resource Editor.  This is feature parity with 4.2.

**Comment by dwuthrich** [comment]
Also, the card manager needs to support ordering of cards
